### ISSUE TITLE
Collaborate possibility

### ISSUE THREAD

**Issue by kennetpostigo** [glennsl/reason-syntax-guide#1]
Hey I wrote a guide for reason and I think a lot of it has to do with syntax. I would really welcome any additions and suggestions you might be working on. If you are interested and willing to add to it I would open it up so that you can have free reign to edit and add as you want. The best way to collaborate would be on gitbooks.io and what i'm using currently. Is this something you would be interested in?

**Comment by glennsl** [comment]
Certainly! If you look at what I've checked in now, you'll hopefully get a decent idea of what I'm going for. I want to provide documentation sources for each item, and your introduction is a great source! If I find the description of something in your guide lacking, I'd be happy to contribute. And I hope you'll add to this if you come across anything missing as well.

**Comment by kennetpostigo** [comment]
Yes! I'd love to consolidate resources. I've been toying with the idea of adding comprehensive documentation to many things reason. Creating a Reason Developer Network, similar to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) . I think above anything else this is what people will likely need. Covering things like tooling (esy, rebel, bucklescript, reason-js), libraries like Lwt and Async, etc. I would love to collaborate to build this out with you.

Do you have a gitbook account? If so can you give me your email so I can give you owner rights? So that you can edit and add freely!
